### DISCUSSION TITLE
Sub-domain in vercel

### DISCUSSION THREAD

**Discussion by tommyfm123** [vercel/community#6696] [Help]
I'm developing a website for a client, and he wants to add this website as a subdomain to an external domain that he owns. He is asking for an A record, which Vercel does not provide, or at least I haven't found any option to obtain it. How can I solve this problem? My client's domain is external to Vercel, so this will mainly be for redirection.

**Comment by versecafe** [comment]
@tommyfm123 a CNAME record pointing to Vercel like this 

<img width="1069" alt="image" src="https://github.com/vercel/community/assets/147033096/ff9a9c4a-296d-425a-87c2-89a864038a5d">

And then you can implement the domain like this

<img width="1320" alt="image" src="https://github.com/vercel/community/assets/147033096/5e4a9a80-3c36-4a10-ae16-16b591bf58c7">

**Reply by tommyfm123** [reply]
I can't see that those options. 

All I can see is this

![image](https://github.com/vercel/community/assets/106606402/08259cd6-f57f-445f-860d-5688428e8d5c)

**Reply by versecafe** [reply]
Have you added the Vercel verification record to your DNS?

**Reply by tommyfm123** [reply]
No, I don't know how to do that.

**Reply by versecafe** [reply]
What's your DNS provider, I can find the documentation for you
